src/test/java/finalTask/config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "runner": "saucelabs",
+  "local": {
+    "browser": "firefox"
+  },
+  "selenoid": {
+    "url": "localhost",
+    "port": "4444"
+  },
+  "saucelabs": {
+    "url": "https://ondemand.us-west-1.saucelabs.com:443/wd/hub",
+    "username": "KirillNaumenko",
+    "accesskey": "f3201ecc-281c-45e8-ada8-e6ef100b542f"
```

Comment:
just want to tell you that it is a bad practice to store various credentials in a repo. It is better to store it in environment variables, on the machine where the tests are run, or in the CI/CD tool ,etc

Reply:
yes. But for now i dont have any vault or virtual machine to store credentials somewhere else :)